docs/imputation.md

```diff
@@ -1,5 +1,8 @@
 # Imputation
 
+## Imputation Disclaimer: Currently we are noticing issues related to the imputation process. We are noticing for some PHGs there are very low mapping rates(~20% WGS reads).  These low mapping rates are likely contributing to some severe errors during the path finding steps. We believe we have an idea of some initial fixes and are working to implement.  We will update the documentation as soon as we have a solution.  Users can still build a PHG without running into this issue. We apologize for any inconvenience this may cause.
```

Comment:
Some suggested changes:

I would remove the "We are noticing" in the second sentence - it is too repetitious.  Second sentence should start with "Some PHGs ..."

The sentence that begins with "We believe we have an idea of some initial fixes " seems too wordy and sounds too tentative.  I'd reword to something like "We have identified issues and are implementing initial fixes"

I think the second to last sentence ("Users can still build a PHG without running into this issue") might be clearer if you said  "This issue does not affect the building of the PHG, only imputation runs.  Existing PHG builds should be fine"

Reply:
Similar to the prior statement but with mkdocs-flavored markdown:

```
!!! danger "Imputation disclaimer"
    We are experiencing issues with imputation, where some PHGs have low 
    mapping rates (~20% with WGS reads), causing errors in the path-finding steps. 
    We are working on fixes and will update the documentation once a solution 
    is available. 
     
    You can still build a PHG without encountering this issue. We apologize 
    for the inconvenience.
```

Reply:
@btmonier Is it better to use mkdocs form or one that looks good on the ReadMe from the GitHub.  I updated it to match this form, but it looks bad from the GitHub page.  Any ideas?

Reply:
@zrm22  - It's because mkdocs-material and GitHub markdown use different syntax on how to build admonition blocks. I use GitHub flavored markdown for the README since that is not rendered for the site.